lib/active_fedora/delegated_attribute.rb

```diff
@@ -2,105 +2,12 @@ module ActiveFedora
   # Represents the mapping between a model attribute and a field in a datastream
```

Comment:
We need to update this comment.  This no longer represents a field in a stream, but a field in any object